docs/config.md

```diff
@@ -78,8 +78,8 @@ Propagators allow applications to share context. See [the OpenTelemetry specific
 for more details.
 
 | Environment variable | Description                                                                                                                                                                                                                                                                                         | Default value          |
-|----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------|
-| `OTEL_PROPAGATORS`   | Comma-separated list of propagators. Supported options: `tracecontext`, `baggage`, `b3multi`. See [the OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#general-sdk-configuration) for more details. | `tracecontext,baggage` |
+|----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------|
+| `OTEL_PROPAGATORS`   | Comma-separated list of propagators. Supported options: `tracecontext`, `baggage`, `b3multi`, `b3`. See [the OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#general-sdk-configuration) for more details. | `tracecontext,baggage` |
```

Comment:
side note: maybe it would be safer to link to a tag instead of the `main` branch`

Reply:
ff3d107322ae698b08dbc99a38c542bc1cab601d